src/action-creators/importText.ts

```diff
@@ -3,6 +3,7 @@ import { parse } from 'jex-block-parser'
 import he from 'he'
 import { contextOf, convertHTMLtoJSON, head, importJSON, pathToContext, rootedContextOf, strip } from '../util'
 import { ActionCreator, Path } from '../types'
+import { Payload } from '../reducers/updateThoughts'
```

Comment:
This would be clearer as `Payload as UpdateThoughtsPayload` since `Payload` is so generic.